spec/requests/entities_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'Entities', type: :request do
+  before(:example) do
+    @user = User.create(
+      name: 'Zuhaib',
+      email: 'zuhaib@example.com',
+      password: '123456'
+    )
+    @group = Group.create(name: 'Shopping', author_id: @user.id)
+  end
+
+  before do
+    # stub the authenticate_user! method
+    allow_any_instance_of(EntitiesController).to receive(:authenticate_user!).and_return(true)
+    allow_any_instance_of(EntitiesController).to receive(:current_user).and_return(@user)
+  end
+
+  describe 'GET /index' do
+    before(:example) { get group_entities_path(@group) }
```

Comment:
- [ ] Great work you put in writing your tests, kindly check I get the error in the screenshot below when I run test.

![test-error](https://user-images.githubusercontent.com/48371156/221009390-144b5347-3dd1-47c5-883c-a43b06bf1db8.png)